foundations/introduction/how_this_course_will_work.md

```diff
@@ -20,7 +20,22 @@ Most lessons will contain questions that you should be able to answer before mov
 
 Try not to think of The Odin Project, or programming, as a class in school. It's not material you learn all at once to take a test, and then pass or fail. You can think of it as a snowball. You, yourself, are a snowball. You're rolling down a hill full of snow; the further you roll, the more snow will stick to you. Sure, snow will also fall off you, and you'll forget things often, but that's just part of the process. Don't be scared if you get to a project and you feel like you haven't retained or memorized anything. That's natural and happens to everyone. The information will come back to you as you start solving your problems one at a time, relying on Google and the Odin Community for help.
 
-### A note about language
+### A note about tools
+
+We often get questions about why this course does not cover a language or a tool.
+
+The truth is, once you complete this curriculum, you will likely **not need** a course like this or any sort of "tutorial". Odin Project's goal is not to teach you any standalone skills and this it not a tutorial. The main idea is to teach you the *why*, and the mindset.
+
+The hardest thing to learn is the mindset. The core ideas of programming. Because programming isn't just about coding, it is about problem-solving, asking the right questions, and researching. Getting into a position where you are comfortable with the skills will mean that you will also be comfortable at expanding your skillset. Once you have a solid *base*, it will be simpler to expand upon it. This is exactly what The Odin Project is about, developing thore *foundations*.
+
+After completing it, you will likely be in a position where you can simply jump straight into the documentation for whatever thing you are trying to learn.
+Reading the documentation is the the best way to learn any tooling. But it takes skill to get good at it. Many people feel intimidated by the techical jargon, which is understandable. People who write documentations usually assume a background in programming. This is exactly why we give you plenty of material to read on your own.
+
+The skills you will gain from completing the Odin Project will be the foundation that you will be building upon for years and decades to come. If you come out of the coure thinking that you need another course like this one to learn something like Python, then you either don't believe in yourself or you haven't taken away the important ideas that are covered in this coure.
+
+At the end of the day, the skills you learn here are transferrable. Once you know the *idea* and that it is *possible* to do something, actually finding out how to do that thing becomes significant easier and is simply a google search away most of the times.
```

Comment:
Minor fixes here. I feel it looks a little better without the "At the end of the day" but I won't fight you for it.
```suggestion
The skills you learn here are transferable. Once you know the *idea* and that it is *possible* to do something, actually finding out how to do that thing becomes significantly easier and is simply a Google search away most of the times.
```